go/internal/feast/utils.go

```diff
@@ -0,0 +1,96 @@
+package feast
```

Comment:
nit, `utils.go` is smell that this is a kitchen sink - can we name this file something more descriptive?